gatling-maven-plugin/pom.xml

```diff
@@ -47,6 +49,18 @@
 			<groupId>org.apache.maven</groupId>
 			<artifactId>maven-compat</artifactId>
 		</dependency>
+		<dependency>
+			<groupId>org.powermock</groupId>
+			<artifactId>powermock-module-junit4</artifactId>
+			<version>${powermock.version}</version>
+			<scope>test</scope>
+		</dependency>
+		<dependency>
+			<groupId>org.powermock</groupId>
+			<artifactId>powermock-api-mockito</artifactId>
+			<version>${powermock.version}</version>
+			<scope>test</scope>
```

Comment:
Yes, I reworked the stuff quite a few times so I did not see the clear solution anymore. Will introduce the getters and setters required for the variables and then it should be possible to work without powermock